copier/main.py

```diff
@@ -513,6 +517,34 @@ def _render_file(self, src_abspath: Path) -> None:
             dst_abspath.write_bytes(new_content)
             dst_abspath.chmod(src_mode)
 
+    def _render_symlink(self, src_abspath: Path) -> None:
+        assert src_abspath.is_absolute()
+        src_relpath = src_abspath.relative_to(self.template_copy_root)
+        dst_relpath = self._render_path(src_relpath)
+        if dst_relpath is None:
+            return
+        dst_abspath = Path(self.subproject.local_abspath, dst_relpath)
+        if dst_abspath.is_dir():
+            return
+        src_mode = src_abspath.lstat().st_mode
+
+        src_target = readlink(src_abspath)
+        dst_target = self._render_path(src_target)
+        if dst_target is None:
+            return
```

Comment:
imho the symlink should be copied raw always, when `preserve_symlinks == True`.

If it is absolute, it should be copied absolute. If it is relative, then relative. Template maintainers can create the symlink as they want, so it's better to lend them that responsability. Just document it and test it.